tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -25,6 +26,9 @@
 #include <utility>
 #include <vector>
 
+template <class T>
+using SpaceshipType = decltype(std::declval<T>() <=> std::declval<T>());

```

Comment:
Should we remove the `#ifdef __cpp_lib_concepts` from around `compare_three_way_result_t` in `<compare>` - which notably doesn't need concepts - so we can use it here?

Reply:
Hmmm, maybe not right now? I don't feel like muddying this PR more.